src/main/java/no/digipost/monitoring/thirdparty/TimedThirdPartyCall.java

```diff
@@ -64,12 +64,12 @@ public RESULT call(Supplier<RESULT> thirdPartyCall) {
 
         if (AppStatus.FAILED == reportWarnPredicate.apply(returnValue, thrown)) {
```

Comment:
Egentlig ikke din endring, men hvis du har tid, trekk gjerne ut resultatet av funksjonskallet på `reportWarnPredicate` i en variabel. Slik at den ikke kalles to ganger, når AppStatus ikke er FAILED.